packages/vuetify/src/components/VList/VListItem.sass

```diff
@@ -4,6 +4,7 @@
 +theme(v-list-item) using ($material)
   &--disabled
     color: map-deep-get($material, 'text', 'disabled')
+    pointer-events: none
```

Comment:
This is already set on line 38. Non-theme styles shouldn't go in the theme section as they'll be duplicated.

Reply:
So line 38 was override by enter-to at Vmenu.sass line:48. Should I remove this one instead?

![image](https://user-images.githubusercontent.com/5492274/90329276-d737ae00-dfd5-11ea-97f4-7e94a2fe0575.png)

Reply:
Uh yeah that definitely shouldn't be there, I thought it was a browser bug. 

8cc1c719a4ee18b195db2b42182a5700fda72291

Reply:
Updated, please verify.